controllers/profiles/gitops/profile_gitops_test.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func Test_Reconciler_ProdOps(t *testing.T) {
-	workflow := test.GetBaseSonataFlowWithProdOpsProfile(t.Name())
+	workflow := test.GetBaseSonataFlowWithGitOpsProfile(t.Name())
```

Comment:
Is this change correct, as instead of prod profile should be used preview?

Reply:
ops, yup, should be Preview. It's just the naming, thanks @jakubschwan!